generated/nise/session.py

```diff
@@ -0,0 +1,871 @@
+        to other Switch Executive API methods (such as connect,
+        disconnect, connect_and_disconnect, and expand_route_spec)
+        that use route specification strings.
+
+        Args:
+            route_spec_size (list of int): The routeSpecSize is an ViInt32 that is passed by reference into the
+                method. As an input, it is the size of the route spec string buffer
+                being passed. If the route spec string is larger than the string buffer
+                being passed, only the portion of the route spec string that can fit in
+                the string buffer is copied into it. On return from the method,
+                routeSpecSize holds the size required to hold the entire route spec
+                string. Note that this size may be larger than the buffer size as the
+                method always returns the size needed to hold the entire buffer. You
+                may pass NULL for this parameter if you are not interested in the return
+                value for routeSpecSize and routeSpec.
+
+
+        Returns:
+            route_spec (str): The route spec of all currently connected routes and route groups. Route
+                specification strings can be directly passed to connect,
+                disconnect, connect_and_disconnect, or expand_route_spec
+                Refer to Route Specification Strings in the NI Switch Executive Help for
+                more information. You may pass NULL for this parameter if you are not
+                interested in the return value. To obtain the route specification
+                string, you should pass a buffer to this parameter. The size of the
+                buffer required may be obtained by calling the method with NULL for
+                this parameter and a valid ViInt32 to routeSpecSize. The routeSpecSize
+                will contain the size needed to hold the entire route specification
+                (including the NULL termination character). Common operation is to call
+                the method twice. The first time you call the method you can
+                determine the size needed to hold the route specification string.
+                Allocate a buffer of the appropriate size and then re-call the method
+                to obtain the entire buffer.
+
+        '''
+        vi_ctype = _visatype.ViSession(self._vi)  # case S110
+        route_spec_ctype = (_visatype.ViChar * route_spec_size[0])()  # case C080
+        route_spec_size_ctype = get_ctypes_pointer_for_buffer(value=route_spec_size, library_type=_visatype.ViInt32)  # case B550
+        error_code = self._library.niSE_GetAllConnections(vi_ctype, route_spec_ctype, route_spec_size_ctype)
+        errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
+        return route_spec_ctype.value.decode(self._encoding)
+
+    def get_ivi_device_session(self, ivi_logical_name):
```

Comment:
Do we need a way to instantiate a session object from an already open session? And then should this try to use that?

Reply:
Since there is no IVI python API I will remove it